.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ jobs:
   test:
     working_directory: ~/boilerplate-client-angular
     docker:
-      - image: 'circleci/node:10-browsers'
+      - image: 'circleci/node:14-browsers'
```

Comment:
Nice! We were pretty behind on the node version.

One question, since the new version of Angular CLI requires Node v14.15, can we restrict the image here to 14.15 so it doesn't pull a newer version of node 14.* and cause CI to fail?

Reply:
Thank you for the suggestion. I just pushed up a change to enforce the specific version. It looks like it works 👍